cli/common/socket.go

```diff
@@ -91,7 +91,7 @@ func EvalTarantoolConn(conn net.Conn, funcBody string) (interface{}, error) {
 	end
 
 	if err ~= nil then
-		return { success = false, err = err }
+		return { success = false, err = tostring(err) }
```

Comment:
Could be moved into separate patch as completely.

Reply:
See #410